LayoutTests/fast/rendering/render-multi-column-set-invalidate-crash.html

```diff
@@ -0,0 +1,74 @@
+<head>
+<style>
+:not(bgsound) { -webkit-column-rule: solid; column-span: all; }
+.class1 { display: flex; float: right; black; -webkit-user-modify: read-write; }
+* { green; -webkit-column-count: 44; }
```

Comment:
```suggestion
* { column-rule: solid; column-span: all; column-count: 44; }
.class1 { display: flex; float: right; -webkit-user-modify: read-write; }
```

There's no bgsound in the file, so `:not(bgsound)` is just `*`. Although I doubt we need to target all elements on the page. The invalid `black;` and `green;` can be removed.